pycaret/internal/display/display_backend.py

```diff
@@ -177,10 +177,6 @@ def _handle_input(self, obj: Any) -> Any:
         return obj
 
 
-backends = [CLIBackend, JupyterBackend, ColabBackend, DatabricksBackend, SilentBackend]
```

Comment:
Example: the type of `backends` was a list or a dictionary.

In addition to that, it does not have to be a global variable because of its usage, so it should be moved.